code/modules/jobs/job_types/head_of_security.dm

```diff
@@ -32,7 +32,7 @@
 	bounty_types = CIV_JOB_SEC
 
 	family_heirlooms = list(/obj/item/book/manual/wiki/security_space_law)
-
+	rpg_title = "Guard Leader"
```

Comment:
Head of the Guard

Reply:
I purposefully left it as RPGMaker SOUL

Reply:
?

Reply:
its hard to quantify, just know town guard npcs are like a dime a dozen

<details>
  <summary>Spoiler warning</summary>
  
  ## SOUL
  
  ![](https://i.ytimg.com/vi/bESMvBzkSJI/maxresdefault.jpg)
  
</details>

Reply:
what does that have to do with the bad name?